EpsilonLights/Inc/Lights.h

```diff
@@ -55,6 +55,20 @@ typedef struct SigLightsHandle
     uint8_t right;
 } SigLightsHandle;
 
+typedef struct lightsInfo
+{
+    char headlightsOff;
+    char headlightsLow;
+    char headlightsHigh;
+    char rightSignal;
+    char leftSignal;
+    char hazards;
+    char brakes;
+    char bmsStrobe;
+    uint32_t regenBrakeInt;
+    float regenBrakeFloat;
```

Comment:
The `regenBrake` stuff isn't really 'lights information' unless I am misunderstanding. I would probably make another struct called `RegenBrakeInfo`. Also struct names should be PascalCase.